app/views/comments/new.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<h3>Add a Comment!</h3>
+<%= form_for @comment do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :commentor %>
```

Comment:
Matching labels and text fields (or other types of inputs) should have matching names (e.g., both should be called `:name` or `:commentor`).